### DISCUSSION TITLE
How to get syntax-highlighting for '*.pro' file for qmake?

### DISCUSSION THREAD

**Discussion by tevaughan** [NvChad/NvChad#2581] [Q&A] [answered]
I see that there is an old-style 'qmake.vim' and filetype configuration at https://github.com/artoj/qmake-syntax-vim

Is there a standard way to incorporate this into my NvChad-configuration?
It's not obvious to me how to do that.

Or is there a more modern solution?
It did not pop up for me when I searched.

**Comment by siduck** [comment, marked as answer]
@tevaughan  
```lua
{ 'qmake-syntax-vim', lazy=false }
```

**Reply by tevaughan** [reply]
Thank you very much!
That worked!

Not being familiar with convention, I am wondering how the simple 'qmake-syntax-vim' works.
I might have expected the need to use 'artoj/qmake-syntax-vim' (with the 'artoj/' prefix).

Again, thanks!

**Reply by tevaughan** [reply]
When I deleted my ~/.local/share/nvim and started neovim again, it failed to install 'qmake-syntax-vim'.

But changing it to 'artoj/qmake-syntax-vim' in ~/.config/nvim/lua/custom/plugins.lua fixed the problem.

I must have already tried installing 'artoj/qmake-syntax-vim' as a package at some point before asking my question here, but it didn't seem to work.

Hence this thread.
But now everything seems to work.

Your suggestion without the 'artoj/' must have worked because the repo was already checked out under ~/.local/share/nvim/lazy/qmake-syntax-vim.

Thanks again!

**Reply by siduck** [reply]
@tevaughan you are right i forgot to add the repo author name so it should be `artoj/qmake-syntax-vim`
